### DISCUSSION TITLE
How to integrate @next/bundle-analyzer within a nx workspace?

### DISCUSSION THREAD

**Discussion by mcmxcdev** [vercel/next.js#40267] [Help]
I am trying to integrate https://www.npmjs.com/package/@next/bundle-analyzer into a next.js app as part of a nx workspace.

The dependency recommends toggling the analyzer based on `process.env.ANALYZE` env flag which is enabled in a npm script in `package.json`.

With our nx setup, we don't have any `package.json` files (only `project.json`) except for the root one and would like to keep it that way.

Is there a recommended way to make this work?

**Comment by icyJoseph** [comment]
I am not 100% sure but you could define the variable in an ad-hoc way via `ANALYZE=true nx build myapp` or something like that, and it should do the trick.

**Reply by mcmxcdev** [reply]
I guess thats probably the best way and have it by default be false all the time.
